setup.cfg

```diff
@@ -38,13 +38,12 @@ packages =
 include_package_data = True
 install_requires =
     rdflib>=6.1.1
+    pandas>=1.3.5
```

Comment:
I don't think this is a good idea. Just because of this one optional output format, to PANDAS, you've now made SPARQLWrapper always import PANDAS and PANDAS is a large library! This is overkill. 

Can you make this an optional import, the sort of things like `pip install sparqlwrapper[pandas]`. Then people only need to install PANDAS if they actually want to use it.

I don't know how to do that but I know it's possible.

Reply:
Good fix for this issue! Just follow through with requirements.txt changes please!